pop-control.lua

```diff
@@ -0,0 +1,124 @@
+-- Script to control the various population caps as well as use of max-wave and hermit persistently per fortress
+-- by Tachytaenius
+--[====[
+pop-control
+===========
+Controls the various population caps as well as use of max-wave and hermit persistently per fortress
+Intended to be placed within ``onMapLoad.init`` as ``pop-control on-load``
+Available arguments:
+
+- ``on-load`` automatically checks for settings for this site and prompts them to be entered if not present
+
+- ``reenter-settings`` lets you revise settings for this site
+
+- ``view-settings`` shows you the current settings for this site
```

Comment:
This is quite different from other scripts. It might fit in better if it had this syntax:
no params - show current settings
`set <num> <num> <num>` - set settings
`start` - start the script
`stop` - stop the script (though I don't know if you can really stop this script)

Reply:
I don't think there is any sensible definition of stop, but migrating away from UI makes sense.

Reply:
Except, how do we get a prompt on new fort?

Reply:
Resolving as "fine for now" as per discord